From 764ce9e69477b47577fc0231f4a0190b119937f7 Mon Sep 17 00:00:00 2001
From: Joseph Richard <joseph.richard@windriver.com>
Date: Tue, 19 Nov 2019 13:06:43 -0500
Subject: [PATCH] Include starlingx initscripts changes

This is build with build-pkgs --edit initscripts, and then rebased with
git rebase -i --root, with all existing meta-patches squashed into this
commit, and then the new meta-patch created from the output of
git format-patch HEAD~

Signed-off-by: Joseph Richard <joseph.richard@windriver.com>
Signed-off-by: Dongqi Chen <chen.dq@neusoft.com>
---
 SPECS/initscripts.spec | 19 +++++++++++++++----
 1 file changed, 15 insertions(+), 4 deletions(-)

diff --git a/SPECS/initscripts.spec b/SPECS/initscripts.spec
index fb0bd96..a446775 100644
--- a/SPECS/initscripts.spec
+++ b/SPECS/initscripts.spec
@@ -19,7 +19,7 @@ Requires:         gawk                       \
 Name:             initscripts
 Summary:          Basic support for legacy System V init scripts
 Version:          10.00.1
-Release:          1%{?dist}.1
+Release:          1.el8_0.1%{?_tis_dist}.%{tis_patch_ver}
 
 License:          GPLv2
 
@@ -67,6 +67,20 @@ Obsoletes:        %{name}            < 9.82-2
 Patch000:   patch000-fix_escape_sequence_being_output_under_systemd_service_units.patch
 Patch001:   patch001-fix_missing_python3_during_build_phase.patch
 
+Patch1001: support-interface-scriptlets.patch
+Patch1002: relocate-dhclient-leases-to-var-run.patch
+Patch1003: dhclient-restrict-interfaces-to-those-on-c.patch
+Patch1004: support-interface-promisc.patch
+Patch1005: 0001-dhclient-remove-1-arg.patch
+Patch1006: 0001-force-delay-check-link-down.patch
+Patch1007: run-ifdown-on-all-interfaces.patch
+Patch1008: sysconfig-affirmative-check-for-link-carrier.patch
+Patch1009: sysconfig-unsafe-usage-of-linkdelay-variable.patch
+Patch1010: ipv6-static-route-support.patch
+Patch1011: ifup-eth-stop-waiting-if-link-is-up.patch
+Patch1012: ifup-alias-scope.patch
+Patch1013: ifup-alias-check-ipaddr.patch
+
 # Downstream patches -- these should be always included when doing rebase:
 # ------------------
 #Patch100: example100.patch
@@ -266,9 +280,6 @@ fi
 
 # NOTE: /etc/profile.d/ is owned by setup package.
 #       /etc/sysconfig/ is owned by filesystem package.
-%dir %{_sysconfdir}/rc.d
-%dir %{_sysconfdir}/rc.d/init.d
-%dir %{_sysconfdir}/rc.d/rc[0-6].d
 %dir %{_sysconfdir}/sysconfig/console
 %dir %{_sysconfdir}/sysconfig/modules
 %dir %{_libexecdir}/%{name}
-- 
2.7.4

